stac_mjx/compute_stac.py

```diff
@@ -45,11 +45,14 @@ def root_optimization(
     s = time.time()
     q0 = jp.copy(mjx_data.qpos[:])
 
-    # Set the center to help with finding the optima (does not need to be exact)
-    # However should be close to the center of mass of the animal. The "magic numbers"
-    # below are for the rodent.xml model. These will need to be changed for other
-    # models, and possibly be computed for arbitray animal models.
-    q0 = q0.at[:3].set(kp_data[frame, :][12:15])
+    # Set the root_kp_index below according to a keypoint in the
+    # KEYPOINT_MODEL_PAIRS that is near the center of the model, not
+    # necessarily exactly so. The value of 3*18 is chosen for the
+    # rodent.xml, corresponding to the index of 'SpineL' keypoint.
+    # For the mouse model this should be 3*5, corresponding 'Trunk'
+    root_kp_idx = 3 * 18
+    root_kp_range = slice(root_kp_idx, root_kp_idx + 3)
```

Comment:
I'm unsure about using `slice` rather than directly having `q0 = q0.at[:3].set(kp_data[frame, :][root_kp_idx:  root_kp_idx + 3]). Just less readable to me.